migrations/dposv3_migration.go

```diff
@@ -55,6 +69,13 @@ func DPOSv3Migration(ctx *MigrationContext) error {
 
 	// Switch over to DPOSv3
 	ctx.State().SetFeature(loomchain.DPOSVersion3Feature, true)
+	// Set feature in chainconfig Contract
+	var feature cctypes.Feature
+	feature.Status = cctypes.Feature_ENABLED
+	feature.BlockHeight = uint64(chainconfigCtx.Block().Height)
+	if err := chainconfigCtx.Set(featureKey(loomchain.DPOSVersion3Feature), &feature); err != nil {
```

Comment:
Instead of this, write a method in chainconfig contract to enable feature.

Reply:
Done.